src/backend/impl_lmdb/arch_migrator.rs

```diff
@@ -7,6 +7,8 @@
 // under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
 // CONDITIONS OF ANY KIND, either express or implied. See the License for the
 // specific language governing permissions and limitations under the License.
+#![allow(dead_code)] // lots of unused struct members
+#![allow(clippy::upper_case_acronyms)] // things like BRANCH
```

Comment:
Just one thing, could you add TODO or XXX to indicate these are not really intentional but just temporary workaround? (Unless you intend to do the followup today 😜)

Reply:
guess these two specifically won't get addressed anway. It's LMDB code, that goes away when we drop LMDB support fully.